.github/workflows/rust.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         filename: cobertura.xml
         badge: true
         fail_below_min: true
-        thresholds: '50 85'
+        thresholds: '70 85'
```

Comment:
I think we should go to 85 and let builds fail due to coverage issues now rather than later.